smarts/core/smarts.py

```diff
@@ -449,6 +449,7 @@ def teardown(self):
 
     def destroy(self):
         self.teardown()
+        self._traffic_sim.destroy()
```

Comment:
I am not sure whether this is the best place to add this function.

Reply:
This should be fine, although, it might be better to put it after the `self._agent_manager.destroy()`

Reply:
Should be fine.